boards/nucleo-f767zi/Makefile.dep

```diff
@@ -1,5 +1,5 @@
 ifneq (,$(filter netdev_default gnrc_netdev_default,$(USEMODULE)))
-#  USEMODULE += stm32_eth
+  USEMODULE += stm32_eth
```

Comment:
I commented this out to test `exampes/gnrc_networking`, but with the ethernet driver it was hanging during SPI register access, although I set `GNRC_NETIF_NUMOF := 2` (ethernet + at86rf233). But this is not the topic of this PR.